logging/logrus/registry.go

```diff
@@ -229,3 +237,17 @@ func (lr *logRegistry) getLoggerFromMapping(logger string) *Logger {
 	panic("cannot cast log value to Logger obj")
 
 }
+
+// HookConfigs stores hook configs provided by log manager
+// and applies hook to existing loggers
+func (lr *logRegistry) AddHook(hook logrus.Hook) {
+	lr.hooks = append(lr.hooks, hook)
+
+	lgs := lr.ListLoggers()
+	for lg := range lgs {
+		logger, found := lr.Lookup(lg)
+		if found {
+			logger.AddHook(hook)
```

Comment:
Are you sure this works?? Because I don't see implementation for this...

Reply:
There is implementation in logger.go
func (logger *Logger) AddHook(hook lg.Hook)